src/scvmm/HISTORY.rst

```diff
@@ -2,6 +2,8 @@
 
 Release History
 ===============
+1.1.3
+* Add command `az scvmm create-from-machines`.
 
```

Comment:
Adding new cmd would be taken as new feature and should upgrade the minor part of release version, which should be `1.2.0` here.

Please refer to this cli extension versioning schema doc here: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#cli-extension-version-transition-table

Reply:
Have updated to `1.2.0`